unittests/rpc/handlers/impl/FakesAndMocks.h

```diff
@@ -122,7 +122,7 @@ class CoroutineHandlerFake
     }
 
     Result
-    process(Input input, boost::asio::yield_context& yield) const
+    process(Input input, RPCng::Context ctx) const
```

Comment:
probably should match the signature of the original one - consider adding const 😃